e2e/playwright/viewer/tsconfig.e2e.json

```diff
@@ -7,6 +7,9 @@
     "target": "es2017",
     "types": ["jasmine", "jasminewd2", "node", "@playwright/test"],
     "skipLibCheck": true,
+    "paths": {
+      "alfresco/playwright-shared": ["../../../projects/aca-playwright-shared/src/index.ts"]
```

Comment:
Incorrect name at left side, instead "alfresco/playwright-shared should" be "@alfresco/playwright-shared"